### PR TITLE
Adds a rake task to update OpenProject's Gemfile.lock

### PR DESCRIPTION
**Pull request by jonasheinrich**
https://community.openproject.org/work_packages/20975

### CODE REVIEW

File: lib/tasks/translations.rake [hunk 1]
```diff
@@ -184,4 +184,44 @@ namespace :translations do
       File.rename translation_file, OpenProject::Translations::Engine.root.join('config', 'locales', "#{js_translation?(translation_file) ? 'js-' : ''}#{language_key}.yml")
     end
   end
+
+  desc "Update Gemfile.lock to newest Version of OpenProject Translation"
+  task :update_gemfile_lock => :environment do
+    target_path = 'tmp/openproject'
+    branch = ENV['Translation_working_branch']
```
Comment (ulferts) [inline lib/tasks/translations.rake:191]: This one should be in all uppercase.

File: lib/tasks/translations.rake [hunk 2]
```diff
@@ -184,4 +184,44 @@ namespace :translations do
...
+    system "git clone git@github.com:opf/openproject #{target_path}"
+    Dir.chdir target_path do
+      system "git checkout #{branch}"
+
+      # update Gemfile.lock
+      # Calling 'bundle update openproject-translations' would also try to
+      # update Sprockets and that fails.
+      # So I comment it out, bundle, uncomment it and bundle again.
+      # This way only OpenProject-Translations gets updated.
+      # Maybe we can refactor this when the reference to Sprockets is fixed.
```
Comment (ulferts) [inline lib/tasks/translations.rake]: We probably need to update sprockets and not work around the problem. Especially as the problem should be fixed once rails 4 is merge, I wouldn't want to have the work around in there as it is likely to stay and cause trouble later.

So let's rather see how we can update sprockets.

Comment (cratz) [inline lib/tasks/translations.rake]: Or just use `bundle update --source ....` until rails 4 branch finds the way ;)

File: lib/tasks/translations.rake (hunk 1 above)
Comment (jonasheinrich) [inline lib/tasks/translations.rake:191]: This is a remnant from [old times](https://github.com/finnlabs/ci/blob/dev/ci.sh#L316)

**Pull request merged**